Specs/DateTime/Japanese/SetExtractor.json

```diff
@@ -215,18 +215,6 @@
       }
     ]
   },
-  {
-    "Input": "5月9日から2泊予約できますか。",
```

Comment:
Why "2 nights" not tagged?

Reply:
in English only "nights" is extracted, but in Japanese there is no distinction between singular and plural, so this kind of expressions cannot be recognized as Set (we encountered the problem already when working on Set for expressions like "mornings", "mondays"...).

Reply:
in the model is now extracted as "2 nights" (duration) as in English

Reply:
What would be the closest Set in Japanese?

Reply:
When working on Set, we replaced "mornings", "mondays"... with "each morning", "every monday" etc. 
But in this particular sentence such substitution does not make sense.